app/ui/src/app/store/step/step.store.ts

```diff
@@ -94,10 +93,15 @@ export class StepStore {
       description: "Sends a message to the integration's log",
       configuredProperties: undefined,
       properties: {
-        message: {
+        expression: {
           type: 'string',
           displayName: 'Log Message',
-          required: true,
+          required: false,
```

Comment:
Is a log step useful without a user-supplied message?

Reply:
Yes, in theory the user could just log the body of the message by clicking the checkbox. Does it make sense?

Reply:
Ah, of course, yep!